save-backend/src/main/kotlin/com/saveourtool/save/backend/service/UserDetailsService.kt

```diff
@@ -82,17 +82,20 @@ class UserDetailsService(
      * @return UserSaveStatus
      */
     @Transactional
-    fun saveUser(user: User, oldName: String?): UserSaveStatus = if (oldName == null) {
-        userRepository.save(user)
-        UserSaveStatus.UPDATE
-    } else if (userRepository.validateName(user.name) != 0L) {
```

Comment:
Stupid bug: if you save new users, you don't validate. 
Usually it can be easily tested by simply creating a new user locally and checking it's functionality, but no one fucking cared 🤣